azimuth/routers/v1/model_performance/confusion_matrix.py

```diff
@@ -40,12 +40,14 @@ def get_confusion_matrix(
     pipeline_index: int = Depends(require_pipeline_index),
     without_postprocessing: bool = Query(False, title="Without Postprocessing"),
     normalized: bool = Query(True, title="Normalized"),
+    preserved_class_order: bool = Query(False, title="Preserved Class Order"),
```

Comment:
Est-ce que ça te gosse les query parameters in `ed`? We could also go without. I don't have a strong preference, as long as it is uniform. Your call. Je voulais juste te faire penser.
```suggestion
    normalize: bool = Query(True, title="Normalize"),
    preserve_class_order: bool = Query(False, title="Preserve Class Order"),
```

Reply:
Oh 100% your suggestion is even better, parfait!

Reply:
Cool! I think I have a slight preference for without `-ed`.

Reply:
Also/or, if you prefer, you can flip the boolean, like `reverse_cuthill_mckee`, with `True` by default (like `normalized`).
```suggestion
    normalized: bool = Query(True, title="Normalized"),
    reverse_cuthill_mckee: bool = Query(True, title="Reorder classes to reduce the bandwidth of the matrix using reverse Cuthill–McKee algorithm"),
```

Reply:
I would keep the same logic as the switch. I thought it would be best for the switch to display 'Keep user-provided class order' or something like that, and to be `False` by default. But the reverse could also work, to display on the switch 'Reorder classes based on reverse Cuthill–McKee algorithm'.

Reply:
Right, as you wish.
By the way, the other switch says `Normalize`, and not `Normalized`, so removing the `d` seems perfect anyway.

Reply:
If we reverse the switch and query parameter, we could also label the switch something short and sweet like "Reorder classes" and add a Tooltip to provide info about how it is reordered. I think that's a (small) plus. And I like that the two switches are on the same side by default. It feels consistent.
![image](https://user-images.githubusercontent.com/8386369/189982065-fd9239c4-948a-4ac3-b1e2-6e03b71144b8.png)

Reply:
Ok, I can make the change

Reply:
Done - I will let you push your changes as well.